nilearn/datasets/description/main.rst

```diff
@@ -0,0 +1,37 @@
+Montreal Artificial Intelligence and Neuroscience conference 2018 datasets
+
+
+Notes
+-----
+This functional MRI datasets is used for teaching how to use
+machine learning to predict age from rs-fmri with Nilearn.
+
+The dataset consists of 50 children (ages 3-13) and 33 young adults (ages
+18-39). This rs-fmri data can be used to try to predict who are adults and
+who are children.
+
+The data is downsampled to 4mm resolution for convenience. The original
```

Comment:
I think we should link to OSF here, too, as it describes the preprocessing and downsampling for anyone interested.